src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
+
+const array = [...rows];
+const copyArray = [...array];
```

Comment:
You are creating two copies of the `rows` array here. This is unnecessary and can waste memory. You can sort the `rows` array directly.